code/modules/power/singularity/singularity.dm

```diff
@@ -446,3 +447,52 @@
 	explosion(src.loc,(dist),(dist*2),(dist*4))
 	qdel(src)
 	return(gain)
+
+/obj/singularity/vomit
+	name = "vomit-charged singularity"
+	desc = "It's a gravitational singularity made out of vomit. There appears to be a giant goose stuck inside."
+	icon = 'icons/obj/gooseularity.dmi'
+	icon_state = "gooseularity"
+	pixel_x = -96
+	pixel_y = -96
+	density = FALSE
+	current_size = 9 //It moves/eats like a max-size singulo, aside from range.
+	contained = 0 //Are we going to move around?
+	dissipate = 0 //Do we lose energy over time?
+	move_self = 1 //Do we move on our own?
+	grav_pull = 0 //How many tiles out do we pull?
+	consume_range = 5 //How many tiles out do we eat
+	CanSupermatter = FALSE
+	light_power = 0.7
+	light_range = 7
+	light_color = rgb(0, 255, 0)
+
+/obj/singularity/vomit/consume(atom/A)
+	if (istype(A, /obj/effect/decal/cleanable/vomit))
+		return
+	var/turf/spawnloc = isturf(A) ? A : get_turf(A)
+	var/turf/Tgoose = get_turf(src)
+	playsound( Tgoose, 'sound/effects/splat.ogg', 50, 1)
+	new /obj/effect/decal/cleanable/vomit(spawnloc)
+	..()
+
+/obj/singularity/vomit/expand(force_size = 0)
+	return //this prevents the gooseulo from reverting to a stage 1 immediately upon spawning
+
+/obj/singularity/vomit/singularity_act()
+	return //prevents gooseulo from violently exploding as soon as it's spawned
+
+/obj/singularity/vomit/attack_hand(mob/user) //i know this looks unbelievably hacky but it's the only way i can get the vomitgoose singularity to eat mobs
+	..()
+
+/obj/singularity/vomit/attack_paw(mob/user)
+	..()
+
+/obj/singularity/vomit/attack_alien(mob/user)
+	..()
+
+/obj/singularity/vomit/attack_animal(mob/user)
+	..()
+
+/obj/singularity/vomit/attackby(obj/item/W, mob/user, params)
+	..()
```

Comment:
???

Reply:
it won't eat mobs without these interactions (from my testing anyways)

Reply:
it should be calling inher by default wtf